website/src/components/datasources-illustration/index.tsx

```diff
@@ -0,0 +1,160 @@
+import { ReactNode } from 'react';
+import { cn, Heading, PRODUCTS, Stud } from '@theguild/components';
+
+export function DatasourcesIllustration({ className }: { className?: string }) {
+  return (
+    <div
+      className={cn(
+        'flex text-white bg-green-1000 gap-8 xl:gap-[72px] py-6 px-4 lg:p-8 lg:py-16 xl:p-[72px] rounded-3xl',
+        className,
+      )}
+    >
+      <div className="[@media(width>=760px)]:w-[400px]">
+        <Heading as="h3" size="md">
+          Connect datasources
+        </Heading>
+        <p className="mt-4 text-white-800">
+          Transform non-GraphQL services into GraphQL-ready interfaces with Mesh, enhancing API
+          consistency.
+        </p>
+        <ul className="flex flex-col gap-6 lg:gap-8 mt-10 lg:mt-12">
+          {[
+            {
+              icon: <TiltMeterIcon />,
+              text: 'Automate the creation of type-safe GraphQL APIs from any data source, ensuring reliability and developer efficiency.',
+            },
+            {
+              icon: <PaperclipIcon />,
+              text: 'Enhance data sources by integrating additional data with full type safety, using Mesh.',
+            },
+            {
+              icon: <ScanLine />,
+              text: 'Extend the capabilities of your schema with Mesh by mocking, caching, and transforming data seamlessly.',
+            },
+          ].map(({ icon, text }) => (
+            <li key={text} className="flex items-start gap-4">
+              <Stud>{icon}</Stud>
+              <p className="text-white-800">{text}</p>
+            </li>
+          ))}
+        </ul>
+      </div>
+      <div className="flex-1 flex justify-center [@media(width<760px)]:hidden">
+        <Illustration className="max-w-[600px]" />
+      </div>
+    </div>
+  );
+}
+
+function Illustration({ className }: { className?: string }) {
+  return (
+    <article
+      className={cn('grid grid-cols-8 [grid-template-rows:repeat(5,auto)] h-[508px]', className)}
+    >
+      <SmallNode className="self-end col-span-2">Mobile App</SmallNode>
+      <SmallNode className="self-end col-span-2 col-start-4">Web App</SmallNode>
+      <SmallNode className="self-end col-span-2 col-start-7">Node.js Client</SmallNode>
+      <Diagonal variant="left" className="col-start-2 row-start-2 col-span-2" />
+      <Diagonal variant="middle" className="col-start-5 row-start-2" />
+      <Diagonal variant="right" className="-col-end-2 row-start-2 col-span-2 justify-self-end" />
+      <div className="col-start-3 col-span-4 row-start-3 flex justify-center">
+        <div className="relative z-10 items-center justify-center gap-4 rounded-2xl p-4 xl:gap-4 xl:py-[22px] bg-gradient-to-br from-[#4F6C6A] to-[#004540] transition-colors duration-500 [&>svg]:shrink-0 lg:px-12 xl:px-16 py-6 backdrop-blur-sm flex flex-col text-center">
```

Comment:
```suggestion
        <div className="relative z-10 items-center justify-center gap-4 rounded-2xl p-4 xl:gap-4 xl:py-[22px] bg-gradient-to-br from-[#4F6C6A] to-[#004540] transition-colors duration-500 lg:px-12 xl:px-16 py-6 backdrop-blur-sm flex flex-col text-center">
```
I think you can add shrink-0 directly on `<PRODUCTS.MESH.logo`